playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -17,6 +17,18 @@
     - always
     - run
 
+# This is an arbitrary UUID value that's passed to the libvirt daemonset setup job for configuring a libvirt secret XML file
+- name: Create Libvirt ceph cinder secret uuid
+  shell: "uuidgen > {{ socok8s_libvirtuuid }}"
+  args:
+    creates: "{{ socok8s_libvirtuuid }}"
+  delegate_to: localhost
```

Comment:
Do we want to generate UUID with each run of that play in same environment? Is it okay to have different generated UUID values and used if play is re-run ?

If its okay to have different UUID with each run, then this logic is fine. We just need to check if its okay and also will it trigger site update anyway if that's the only change ?

Reply:
I don't think it makes sense to regenerate it every time.  yes, when the uuid changes, the site is considered changed, pretty expensive.

Reply:
"creates" will skip the action if the referenced file already exists, so as long as {{ socok8s_workspace }}/libvirt.uuid hasn't been removed since the last run, the UUID will never be regenerated. Each subsequent run will refer back to that file and reuse the same UUID.